app/controllers/skills_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SkillsController < ApplicationController
+
+  # makes debugging using Postman easier. And I don't see why we use CSRF for this form
+  skip_before_action :verify_authenticity_token
```

Comment:
you used to be able to log into the form to save your profile, which is likely why that was there.